galleries/tutorials/artists.py

```diff
@@ -156,8 +156,8 @@ class in the Matplotlib API, and the one you will be working with most
 # (the standard white box with black edges in the typical Matplotlib
 # plot, has a ``Rectangle`` instance that determines the color,
 # transparency, and other properties of the Axes.  These instances are
-# stored as member variables :attr:`Figure.patch
-# <matplotlib.figure.Figure.patch>` and :attr:`Axes.patch
+# stored as member variables :attr:`!Figure.patch
+# <matplotlib.figure.Figure.patch>` and :attr:`!Axes.patch
 # <matplotlib.axes.Axes.patch>` ("Patch" is a name inherited from
```

Comment:
```suggestion
# stored as member variables :attr:`!Figure.patch` and
# :attr:`!Axes.patch` ("Patch" is a name inherited from
```

The explicit targets are not needed when we don't link and actually would render verbatim. Compare this

![grafik](https://github.com/user-attachments/assets/5b19d8b1-4206-473e-bb53-0879be3b4af8)

to the original

![grafik](https://github.com/user-attachments/assets/00468917-750a-4e3b-b547-ee9298f09b72)

Please update all other occurences accoringly as well.

Ping @story645. This is something to be aware of for similar PRs.